### DISCUSSION TITLE
ScriptReference/UIElements.InspectorElement.FillDefaultInspector

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#92] [Notes]
# ScriptReference/UIElements.InspectorElement.FillDefaultInspector

https://docs.unity3d.com/ScriptReference/UIElements.InspectorElement.FillDefaultInspector.html

**Comment by vertxxyz** [comment]
The initialisation of `PropertyField` elements is deferred until another time after the element has been added to the hierarchy. I find the root elements are added, but beyond that nothing is initialised.
Calling Bind doesn't create the nested elements. My personal workaround is an editor coroutine that queries for property fields and loops over them until all fields are bound.
